docs/markdown/Unit-tests.md

```diff
@@ -206,6 +206,11 @@ name(s), the test name(s) must be contained in the suite(s). This
 however is redundant-- it would be more useful to specify either
 specific test names or suite(s).
 
+Since version *1.7.0*, you can pass `--slice i/n` to split up the set of tests
```

Comment:
Various locations need to be updated to say 1.8.0 instead.

Reply:
Fixed :+1: